terraform/ecr_repositories/variables.tf

```diff
@@ -15,7 +15,7 @@ variable "repositories" {
   description = "Defines the repositories to create"
   type        = set(string)
   default = [
-    "bookstore-quarkus",
+    "bookstore-quarkus-reactive",
     "bookstore-quarkus-sync",
```

Comment:
I was thinking about it. Do you think it makes sense renaming the sync variant to blocking?

Reply:
We could do it in a future PR. I would keep this one to only renaming the reactive one as it is already part of my other PRs.